src/steering.rs

```diff
@@ -152,9 +153,9 @@ impl Default for SteeringHost {
             cur_velocity: Vec2::ZERO,
             steering: Vec2::ZERO,
 
-            max_velocity: 250.0,
-            max_force: 150.0,
-            mass: 4.0,
+            max_velocity: MaxVelocity(250.0),
```

Comment:
You cannot query components from inside another component, in my opinion it's better to leave these as `f32`'s

Reply:
Make sense

Reply:
done

Reply:
done